packages/generator/src/generator-utils.ts

```diff
@@ -15,30 +15,65 @@ function hasCapability(object: any, capability: string) {
   return !(capability in object) || object[capability] === 'true';
 }
 
+/**
+ * Checks if the 'sap:deletable' property is present on the given input.
+ * @param entity - Object to be checked
```

Comment:
We have fullstops at the end of parameter descriptions almost everywhere, for consistency it would be nice to have it the same everywhere (although this isn't part of the official docs, right).
I think this is even in the style guide.